ShowPot/ShowPot/Common/Font/CustomFont.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Font.swift
+//  ShowPot
+//
+//  Created by Daegeon Choi on 6/16/24.
+//
+
+import Foundation
+
+// MARK: 커스텀 폰트 관리
+
+/// 커스텀 폰트 종류
+enum CustomFont: String {
+    case pretendard = "Pretendard"
+    case oswald = "oswald"
+}
```

Comment:
```suggestion
enum CustomFont: String {
    case pretendard = "Pretendard"
    case oswald = "Oswald"
}
```
CustomFont가 oswald인 경우에 폰트 이름이 `Oswald-[폰트스타일]`이런식으로 되있어서 저 부분 수정해야할꺼같아요 !!